layouts/shortcodes/install-dotnet.html

```diff
@@ -0,0 +1,3 @@
+<p class="mt-4">
+    Install <strong><a href="https://dotnet.microsoft.com/download" target="_blank">.NET Core 3.0</a></strong>.
```

Comment:
@CyrusNajmabadi, I added this file which contains instructions on installing .NET Core (otherwise Hugo was segfaulting when building the site 😉). Do we need to tweak/embellish these .NET install instructions at all?

For reference, these are the instructions we have for Node.js and Python:
 - https://github.com/pulumi/docs/blob/master/layouts/shortcodes/install-node.html
 - https://github.com/pulumi/docs/blob/master/layouts/shortcodes/install-python.html

Reply:
I tweaked it slightly to say `.NET Core 3.0 SDK or later`